zwave_js_server/client.py

```diff
@@ -146,9 +147,18 @@ def connected(self) -> bool:
         """Return if we're currently connected."""
         return self.state == STATE_CONNECTED
 
-    async def async_send_json_message(
-        self, message: Dict[str, Any], message_id: str = None
-    ) -> None:
+    async def async_send_command(self, message: Dict[str, Any]) -> dict:
+        """Send a command and get a response."""
+        future = self._loop.create_future()
```

Comment:
I think we need to type the future. I'm not sure what's the best practice for that. I've been looking around but not found a good answer yet. Maybe use `Awaitable`?

Reply:
This may be relevant:
https://www.python.org/dev/peps/pep-0585/

Maybe:

```py
future: Awaitable[dict] = loop.create_future()

# or

future: Future[dict] = loop.create_future()
```

Reply:
Fixe.d